guide/16-data-engineering-guides/part1_introduction_to_dataengineering.ipynb

```diff
@@ -0,0 +1,1637 @@
+{
```

Comment:
<p>In the first paragraph, we need to say that SeDF is a capability that is added to a regular Pandas DataFrame <strong>by the ArcGIS API for Python.</strong></p><p> </p><p>In the last sentence of this paragraph, we can add that what is shown below is just a quickstart to SeDF and in subsequent pages of the guide, SeDF will be dealt with in a detailed manner</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/842/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>